package.json

```diff
@@ -15,7 +15,12 @@
     "lint": "turbo lint",
     "test": "pnpm --filter=exchange-sdk test"
   },
+  "dependencies": {
+    "@ledgerhq/hw-app-exchange": "0.6.0-nightly.0",
+    "protobufjs": "^7.3.2"
+  },
   "devDependencies": {
+    "@rollup/plugin-json": "^6.1.0",
```

Comment:
Do we need this?

Reply:
for now, no, but it's just the rollup json plugin for TS which is good to have in any project, we will definitely need it in the future, it is a dev dependency so it does not bloat the app